CHANGES

```diff
@@ -3,14 +3,64 @@ Release 1.15 (development)
 
 .. rubric:: Enhancements
 
+* Support sending a ``RAUC-System-Version`` HTTP header in the initial
+  streaming/inspect request.
```

Comment:
Sending HTTP headers in all initial streaming requests is new in general, isn't it?